File: reidai-7.py
def main():
    N = int(input())
    a = list(map(int, input().split()))
    a.sort()
    result = 0
    for i in range(N):
        result += a[i] if i % 2 == 0 else -a[i]
    print(result)

if __name__ == '__main__':
    main()

File: reidai-7.ts
function main(): void {
  const line1: string = '3';
  const line2: string = '2 7 4';
  const N: number = Number(line1);
  const array: Array<number> = line2.split(' ').map(x => Number(x));

  array.sort((a, b) => b - a);

  let result: number = 0;
  for (let i = 0; i < N; ++i) {
    i % 2 ? result -= array[i] : result += array[i];
  }
  console.log(result);
}

main()
